students/baker_glenn/snakify_lesson_4/factorial.py

```diff
@@ -0,0 +1,8 @@
+# script to calculate the factorial
+print("enter a number")
+number = int(input())
+
+result = 1
+for i in range(number):
```

Comment:
You can change range(number) to range(1, number + 1)  and result *= (i+1) on result * = i